vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerViewEvents.kt

```diff
@@ -32,6 +32,8 @@ sealed class MessageComposerViewEvents : VectorViewEvents {
     data class JoinRoomCommandSuccess(val roomId: String) : SendMessageResult()
     class SlashCommandError(val command: Command) : SendMessageResult()
     class SlashCommandUnknown(val command: String) : SendMessageResult()
+    class SlashCommandNotSupportedInThreads(val command: String) : SendMessageResult()
```

Comment:
Maybe `command` could be of type `Command` here, since this is a valid command.

Reply:
Have you seen this comment?

Reply:
Yes, will update accordingly

Reply:
Done!